composer.json

```
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=7.4",
-        "acquia/acquia_cms": "^1.3.2",
+        "acquia/acquia-cms-starterkit": "^1.0",
```

The `.0` is superfluous.
```suggestion
        "acquia/acquia-cms-starterkit": "^1",
```

---

Done.